dnslol/dnslol.go

```diff
@@ -256,13 +372,30 @@ func Start(e Experiment, names <-chan string, wg *sync.WaitGroup) error {
 		}
 	}()
 
+	// Connect to the database
+	db, err := sql.Open("mysql", dsn)
+	if err != nil {
+		return err
+	}
+	db.SetMaxOpenConns(maxConns)
+	// Reuse connections forever
+	db.SetConnMaxLifetime(0)
```

Comment:
I thought you were going to remove this line?

Reply:
Yup, I removed it on the copy of the tool I was using on a Droplet and forgot to do it in my git source tree locally. D'oh.

Removed in 526186a